### ISSUE TITLE
Can you please make an official statement on the state of this extension?

### ISSUE THREAD

**Issue by ACCIAI0** [Unity-Technologies/vscode-unity-debug#207]
It's broken. Let's face it, since 3.0.1 it doesn't even work reliably on older versions of Unity, let alone 2020.x onward. It's just dead at this point and even though the README explicitly says this extension is not officially supported from Unity it doesn't say whether it has been abandoned, deprecated or anything. It is just blatantly ignored without a real alternative out there. You don't want to support or update it anymore? Fine. But let the community know. You don't want Visual studio code to be 100% compatible with Unity? Fine. But. Let. Us. Know. Because if I am forced to use either Visual Studio or Rider (and pay for them) I'd gladly like to know for sure from an official source.

Edit.: I know Visual Studio Community is free, but only up to a certain dimension of the company and it's way slower than VS code.

**Comment by xoofx** [comment]
The current status is that our teams don't have the bandwidth and expertise to maintain such project, so this extension will be deprecated (#206).

We are in discussions with Microsoft .NET Teams to see what we could do about it but we have currently nothing concrete to share about it.

**Comment by konistehrad** [comment]
Could the repository be marked read-only if it's truly going away, or at least a note added to the top of the README explaining the current state of the project?

**Comment by xoofx** [comment]
> Could the repository be marked read-only if it's truly going away, or at least a note added to the top of the README explaining the current state of the project?

Yes, we plan to do it soon. We would like to make a proper announcement on the forum in the coming weeks.

**Comment by patricknasralla** [comment]
Hi, it is now almost the end of 2022 and there still appears to be no official word on the status of this. The Unity documentation still lists VSCode as a supported IDE, even on the latest version: https://docs.unity3d.com/2023.1/Documentation/Manual/ScriptingToolsIDEs.html

Additionally, that page states that the debugger is still usable but this is clearly not the case any longer. 

I want to stress I'm not writing here to demand support for VS Code (although it should be noted that there are a large group of developers that do in fact use it), that is up to Unity to decide. 

My issue is that the information required to make a decision about which IDEs are compatible is out of date and *needs* to be updated! This leads to considerable wasted time in trying to set up VS Code as the IDE before stumbling onto this thread.

**Comment by xoofx** [comment]
> Hi, it is now almost the end of 2022 and there still appears to be no official word on the status of this. The Unity documentation still lists VSCode as a supported IDE, even on the latest version:

We made an post [here](https://forum.unity.com/threads/update-on-the-visual-studio-code-package.1302621/).

> My issue is that the information required to make a decision about which IDEs are compatible is out of date and needs to be updated! This leads to considerable wasted time in trying to set up VS Code as the IDE before stumbling onto this thread.

Agreed. We will update this information hopefully this month.

**Comment by patricknasralla** [comment]
Thanks for your reply, I can’t seen that thread and it’s very helpful. Much appreciated!

**Comment by stonstad** [comment]
Please update/fix Unity documentation which incorrectly lists VS Code as a viable debugger. 
 https://docs.unity3d.com/2023.1/Documentation/Manual/ManagedCodeDebugging.html.

**Comment by bn-l** [comment]
> > Could the repository be marked read-only if it's truly going away, or at least a note added to the top of the README explaining the current state of the project?
> 
> Yes, we plan to do it soon. We would like to make a proper announcement on the forum in the coming weeks.

Please make a statement in the readme and mark this as archived.

**Comment by BagelSeasoning234** [comment]
That's a shame. Does anyone know of any other viable Unity IDEs on linux besides Rider?
